app/views/users/noticed_notifications/index.html.erb

```diff
@@ -11,9 +11,13 @@
       <span id="all-notifications" class="active notifications-tab" data-action="click->notifications#activeAllNotifications"><%= t("users.notifications.all") %></span>
       <span id="unread-notifications" class="notifications-tab" data-action="click->notifications#activeUnreadNotifications"><%= t("users.notifications.unread") %></span>
     </div>
-    <div>
-      <%= link_to t("users.notifications.mark_as_read_button"), mark_all_as_read_url(current_user), class: "anchor-text" %>
-    </div>
+
+    <% if @unread.count != 0 %>
```

Comment:
```suggestion
    <% if @unread.positive? %>
```

Reply:
it looks better now... thank you! shall I commit the changes?

Reply:
yes, could you also please remove the extra lines mentioned by hound!